src/components/BaseLayerSwitcher/BaseLayerSwitcher.scss

```diff
@@ -0,0 +1,74 @@
+.rs-base-layer-switcher {
+  position: relative;
+  height: 80px;
+  width: 115px;
+  transition: 250ms width;
+  overflow: hidden;
+
+  &.open {
+    width: 360px;
+  }
+
+  .rs-base-layer-switch-layers {
+    display: flex;
+    overflow: auto;
+    height: 100%;
+    width: 360px;
+
+    .rs-base-layer-switch-button {
+      position: relative;
+      height: 85%;
+      min-width: 100px;
+      margin: 4px;
+      overflow: hidden;
+      border: 2px solid white;
+      background-color: gray;
+
+      &.open {
```

Comment:
i think this one is useless. hover and active css can be applied without it.

Reply:
I only want the border effects when the switcher is open, if I remove it the border will be red also when closed

Reply:
ok